src/pages/delegates/delegates.ts

```diff
@@ -162,19 +162,19 @@ export class DelegatesPage {
   ionViewDidEnter() {
     this.currentNetwork = this.arkApiProvider.network;
     this.currentWallet = this.userDataProvider.currentWallet;
-
+    const NUM_ACTIVE_DELEGATES = 51
```

Comment:
Since this is a constant which is almost application specific, I'd move it to here:

https://github.com/ArkEcosystem/ark-mobile/blob/master/src/app/app.constants.ts